python/py-tomlkit/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-tomlkit
+version             0.5.8
+revision            0
+categories-append   devel
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+python.versions     37 38
+
+maintainers         {gmail.com:davidgilman1 @dgilman} openmaintainer
+
+description         Style preserving TOML library
+long_description    ${description}
+
+homepage            https://github.com/sdispater/tomlkit
+
+checksums           rmd160  594810a362d5ba3bca3d1154ea11fcf923f12574 \
+                    sha256  32c10cc16ded7e4101c79f269910658cc2a0be5913f1252121c3cd603051c269 \
+                    size    66567
+
+post-destroot {
+   # https://github.com/sdispater/tomlkit/issues/69
+   delete ${destroot}${python.pkgd}/tests
+}
+
+if {${name} ne ${subport}} {
+    depends_build-append port:py${python.version}-setuptools
```

Comment:
please align with 4 spaces

Reply:
in fact, does it need ```setuptools``` - I don't see it mentioned/used in the ```setup.py``` file

Reply:
I took out the setuptools dependency and was able to get this to build correctly.  No idea why the other sdispater modules required it and this one doesn't, I wonder if we're going to see this one come around when he does another release.